stacker/__init__.py

```diff
@@ -1 +1,25 @@
+from stacker.plan2 import Step, build_plan
+
 __version__ = "1.1.4"
+
+
+def plan(description=None, action=None,
```

Comment:
What's the plan going forward with this? We probably can't name this function "plan" if we move `plan2` to plan.

Reply:
The plan (no pun intended) would be to replace `plan.py` with `plan2.py` before merging, unless we want to keep the original `plan.py` around for backwards compatibility. I only went with `plan2.py` because overwriting `plan.py` in the PR would make reviewing it pretty hard, since it's a re-write.

Reply:
Right, but what about the `plan` method?  If you have a `plan` method in `stacker/__init__.py` and a `stacker/plan` directory, you're going to have conflicts when someone does something like: `from stacker import plan` or `from stacker.plan import *` (maybe not on the last one, I honestly don't know!)

Reply:
Ah, gotcha. I think this `plan` method should actually be moved to `stacker.actions.base`, since that's where it's intended to be used.